nipype/interfaces/freesurfer/utils.py

```diff
@@ -89,8 +89,8 @@ class SampleToSurfaceInputSpec(FSTraitedSpec):
     subject_id = traits.String(desc="subject id")
     target_subject = traits.String(argstr="--trgsubject %s",
                      desc="sample to surface of different subject than source")
-    surf_reg = traits.Bool(argstr="--surfreg", requires=["target_subject"],
-                           desc="use surface registration to target subject")
+    surf_reg = traits.String(argstr="--surfreg %s", requires=["target_subject"],
+                           desc="registration surface to target subject (default sphere.reg)")
```

Comment:
in such a case, you may want to explicitly set the default value of the traits to `sphere.reg`

`surf_reg = traits.String('sphere.reg', ..., usedefault=True)`